frontend/test/pytest/test_qnode.py

```diff
@@ -26,12 +26,12 @@
         (1, True),
     ],
 )
-def test_variable_capture(_in, _out, backend):
-    """Test variable capture."""
+def test_variable_capture_multiple_devices(_in, _out, backend):
+    """Test variable capture using multiple backend devices."""
 
     @qjit()
-    def workflow1(n: int):
-        @qml.qnode(qml.device(backend, wires=2))
+    def workflow(n: int):
+        @qml.qnode(qml.device("lightning.qubit", wires=2))
```

Comment:
Looks good but maybe just make that a separate test? 😅

Reply:
I didn't create a new test because we would end up having a duplicate test for the case `backend="lightning.qubit"`.

Reply:
That's the problem with your other test as well no, since it uses the same device when `backend="lightning.qubit"` it's not testing anything useful 😅 But no big deal either way 👍